### DISCUSSION TITLE
Dynamically create Collections based on Relation Graph

### DISCUSSION THREAD

**Discussion by tadly** [meisnate12/Plex-Meta-Manager#374] [Q&A]
Don't think this is possible yet and haven't found someone else asking the same so here I go.

What I would like is to have Collections automatically be created for everything I own based on the relation graph.

For example:
Let's say I own most of "Sword Art Online" (https://anidb.net/anime/10022/relation/graph).

I would like a collection named "Sword Art Online" (name derived from the "root" show/movie) to be created containing everything referenced in the collection graph. (just like the existing funktionality)
This same logic would then be applied for every show/movie I own so I don't have to define each collection manually.

Is this maybe already possible? If not, would it be possible to implement?

**Comment by meisnate12** [comment]
Make sure to check the wiki you can use anidb_relation already https://github.com/meisnate12/Plex-Meta-Manager/wiki/AniDB-Builders#anidb-relation

**Reply by tadly** [reply]
Yes but this requires me to explicitelly create a collection with the name "Sword Art Online".
I would want Plex-Meta-Manager to **dynamically** create these collections based on my library so I don't have to specifically add them all myself

**Reply by tadly** [reply]
To expand on that and hopefully make it more clear.

Let's say I have 100 Anime in my library.
Each one of those 100 Anime *could* be related to one of the other 99.

I would wan't Plex-Meta-Manager to check **each** anime:
1. If a relation **within my library** exists (based on the anidb relation graph)
2. Create a new collection (it if doesn't already exist) with a name based on the "root" Anime on anidb
3. Add the Anime to said Collection

**Reply by meisnate12** [reply]
Gotcha yea PMM doesn't dynamically create any collections but I was thinking about adding a feature to dynamically create actor collections if they have a certain number of actor credits in your library at some point so dynamically creating AniDB Relation collections is in the same vein. These features will be on the back burner until I finish the GUI cause that's my top priority. Can you make this as a feature request so I don't forget about it.

**Reply by tadly** [reply]
Sure thing. Wanted to make sure this hasn't been talked over before. :)

**Comment by tadly** [comment]
For reference the Feature Request #377 has been created :)
